src/main/java/com/linecorp/armeria/common/logging/ResponseLog.java

```diff
@@ -42,6 +50,18 @@
     AttributeKey<RpcResponse> RPC_RESPONSE = AttributeKey.valueOf(RequestLog.class, "RPC_RESPONSE");
 
     /**
+     * The {@link AttributeKey} of the processed {@link RpcResponse} in its protocol-dependent low-level form.
+     *
+     * <p>For a Thrift response, the value of this {@link Attribute} is an array ({@code Object[]}) whose
+     * length is 2. Its first element is a {@link TMessage} and the second element is a {@link TBase} or
+     * a {@link TException} depending on the type of the {@link TMessage}.
```

Comment:
I guess TException never appears as second element. Only TApplicationException(it's a TException though :p)?